src/content/docs/en/installation/rpi.mdx

```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
   it on another device.
 </Alert>
 
+First, you will need to decompress it.
+
+- On Linux, run `unzstd *FILENAME.img.zst*`, you may need to install the `zstd` package
+- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a Zip file.
+- On Windows 10, use 7zip
```

Comment:
WinZip and WinRar also work

Reply:
I didn't want to include more then one option (for example getting zstd on MacPorts), but I can

Reply:
at least add winrar because it's very very popular